spec/support/flipper.rb

```diff
@@ -0,0 +1,13 @@
+module Flipper
+  RSpec.configure do |config|
+    config.before(:each) do |example|
+      allow(Panoptes).to receive(:flipper).and_return(Flipper.new(Flipper::Adapters::Memory.new))
+      Panoptes.flipper["cellect"].enable
+      Panoptes.flipper["designator"].enable
+      Panoptes.flipper[:remove_complete_subjects].enable
```

Comment:
For some they are always on but they could be off...here we're just saying these are always on for the specs. If it becomes an issue (and i don't think it will) we can change it.